app/src/main/java/com/example/plan2meet/MainActivity.kt

```diff
@@ -4,6 +4,8 @@ import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var day:String
```

Comment:
added variables per the suggestions here:https://kotlinlang.org/docs/basic-syntax.html

Reply:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.
--
Is String the best way to represent a day and month?